### DISCUSSION TITLE
Expand to selected item

### DISCUSSION THREAD

**Discussion by DavidNguyenvp** [dowjones/react-dropdown-tree-select#640] [General]
Hi everyone,

I am currently implementing react-dropdown-tree-select in my project, but I'm encountering an issue. After selecting a child item, my component gets re-rendered, and when I click on the search again, it always resets to the top. I would like to maintain the last selected position.

Could you please advise on how to achieve this?

**Comment by mrchief** [comment]
Check old issues on how to prevent re-render on state change.
